packages/app-studio-toolkit-types/api.d.ts

```diff
@@ -54,6 +54,14 @@ export interface BasToolkit {
    */
   isLCAPEnabled: () => Promise<boolean | undefined>;
 
+  /**
+   * Is opened for action or not
```

Comment:
Need a better description for in the public API.

What does being "open" mean?